internal/network/cidr_validations.go

```diff
@@ -48,6 +48,28 @@ func VerifySubnetCIDR(cidrStr string) error {
 	return nil
 }
 
+func max(x, y int) int {
+	if x > y {
+		return x
+	}
+	return y
+}
+
+func VerifyClusterCidrSize(hostNetworkPrefix int, clusterNetworkCIDR string, numberOfHosts int) error {
+	_, cidr, err := net.ParseCIDR(clusterNetworkCIDR)
+	if err != nil {
+		return err
+	}
+	ones, _ := cidr.Mask.Size()
```

Comment:
Consider renaming `ones` to `clusterNetworkPrefix`

Reply:
OK